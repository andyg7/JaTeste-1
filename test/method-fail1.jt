func int main()
{
	struct car *my_car;

	my_car->0;

	return 0;
}


struct car {
	int price;
	int year;
	string model;

	method void set_model(string s)
	{
		model = s;
	}
};
