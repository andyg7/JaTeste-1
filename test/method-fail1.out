Scanned
Parsed
Fatal error: exception Exceptions.InvalidPointerAccess
